client/my-sites/upgrades/map-domain/index.jsx

```diff
@@ -58,7 +61,7 @@ var MapDomain = React.createClass( {
 
 		const selectedSite = this.props.sites.getSelectedSite();
```

Comment:
This also looks like the perfect place to migrate this to use the `getSelectedSite` selector.